quinn-udp/src/log.rs

```diff
@@ -0,0 +1,21 @@
+#![allow(unused_imports, unused_macros)]
```

Comment:
I'd prefer to just keep all this in `lib.rs` (somewhere near where the previous imports where) instead of in a small separate file.

Reply:
With the recent changes, `log.rs` increased in size. @djc would you still like me to move it into `lib.rs`? Fine by me.

Reply:
Friendly ping @djc. Do you still want me to move the code above to `lib.rs`?

Reply:
Yes, please.